Src/FluentAssertions/Specialized/TaskAssertions.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using FluentAssertions.Common;
+using FluentAssertions.Execution;
+
+namespace FluentAssertions.Specialized
+{
+    /// <summary>
+    /// Contains a number of methods to assert that a task yields the expected result.
+    /// </summary>
+    [DebuggerNonUserCode]
+    public class TaskAssertions
+    {
+        private readonly ITimer timer;
+
+        public TaskAssertions(Task subject, ITimer timer)
+        {
+            Subject = subject;
+            this.timer = timer;
+        }
+
+        /// <summary>
+        /// Gets the <see cref="Task"/> that is being asserted.
+        /// </summary>
+        public Task Subject { get; private set; }
+
+        /// <summary>
+        /// Asserts that the current <see cref="Task"/> will complete within specified time range.
```

Comment:
(nitpick) It's not really a time _range_, is it?

Reply:
done